models/api-models.js

```diff
@@ -0,0 +1,8 @@
+
+
+const fs=require('fs/promises')
+exports.selectAPI = () =>{
+  return fs.readFile('./endpoints.json', "utf-8").then((endpoints)=>{
+    return JSON.parse(endpoints, "Endpoints")
+  })
+}
```

Comment:
We shouldn't have a model for this it should exist in your controller since we are not interacting with the database.